healthcare/api-client/v1/fhir/fhir_resources_test.py

```diff
@@ -61,8 +61,9 @@ def wait_for_operation(operation_name: str):
         .get(name=operation_name)
         .execute()
     )
-    if not operation["done"]:
-        raise OperationNotComplete()
+
+    if not operation.get("done", False):
+        raise OperationNotComplete(operation)
 
```

Comment:
I've tweaked this to assume if "done" is not present, `done = False`. The exception message will contain the operation.